src/components/transaction/CreateTransactionForm.tsx

```diff
@@ -38,83 +40,115 @@ const CreateTransactionForm = ({
   const withdrawMoney = useCreateWithdrawalMutation({ setMessage })
   const transferMoney = useCreateTransferMutation({ setMessage })
 
+  const {
+    register,
+    reset,
+    getValues,
+    formState: { errors, isValid },
+  } = useForm<FormData>({
```

Comment:
If you need to add or take away types you can use a combination of [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) and union symbols to create a new type. Because you have a base global type, you can unionize your global type and an object to add a key to your base type.

```suggestion
  } = useForm<AccountForm & {targetId?: string}>({
```